### ISSUE TITLE
Centralized list of recompiled games/projects

### ISSUE THREAD

**Issue by RyanEwen** [N64Recomp/N64Recomp#72]
It would be really great to have a centralized list of known recomp projects and their authors. Maybe a section can be added to the README with this?

I am so excited about recomps but have no idea where I should go to find out about other (working) projects. I worry that search engines could lead people into piracy-themed forums and topics that aren't quite 'above board' for lists which may not end up being maintained.

**Comment by michaelachrisco** [comment]
I think this already exists in a couple of places. Biggest one that ive seen is this list: https://github.com/command-tab/awesome-n64-development?tab=readme-ov-file#game-decompilation

**Comment by RyanEwen** [comment]
Thanks for the link @michaelachrisco !

I still think it would be great to have a list within this repo as well - specifically listing projects based on N64Recomp in particular. Doesn't mean others can't have their own lists or include the same list within other lists of larger scope, etc.

**Comment by kungfooman** [comment]
I'm interested in static recompilation in general and I would rejoice a list about tested/researched possibilities aswell. For example demakes like Portal64 are interesting to me in combination with compiling it onto e.g. old Raspberry Pi's to turn them into a mini console (ARM 32bit and newer 64bit targets).

**Comment by Extloga** [comment]
> Thanks for the link @michaelachrisco !
> 
> I still think it would be great to have a list within this repo as well - specifically listing projects based on N64Recomp in particular. Doesn't mean others can't have their own lists or include the same list within other lists of larger scope, etc.

As the main contributor to the aforementioned list, I can inform you that my [PR](https://github.com/command-tab/awesome-n64-development/pull/45) for the inclusion of these game-specific projects was ultimately rejected.

**Comment by michaelachrisco** [comment]
@Extloga Might be time to make your own or add it to this repo. Thanks for trying!

**Comment by nonetrix** [comment]
This would be great! The Mario 64 PC port I can't seem to be able to go above 60 FPS for example, unless I attempt to modify the patch and likely mess something up. I am pretty sure this uses frame interpolation though so it should just work right?

**Comment by nonetrix** [comment]
I agree, also is there a recomp for Mario 64? The 60 FPS patch only allows up to 60 FPS unless I tried to modify the patch to go up to 165 in my case, which I might try later. But this seems better, seems like it just interpolates the frames at a engine level

**Comment by Mr-Wiseguy** [comment]
Right now there's no need for a list since Zelda 64: Recompiled is the only finished public static recompilation, but I think a list makes sense once other projects start to release.

I also do understand command tab not wanting to include them on n64.dev, since that's primarily focused on developing for the N64. Decompilation projects are listed on it since they're a great reference for how to write code to run on the N64, but other port projects aren't for the same reason static recompilations don't necessarily fit.

Once more recompilations release, I may make a GitHub wiki with links to all of them if somebody else hasn't curated a list already.

**Comment by Lucoshi** [comment]
How about banjo kazooie games

**Comment by Hellzio** [comment]
No mortal Kombat Trilogy and Killer Instinct Gold? weird,mk needs it since it has very bad graphics if we try to emulate it,as for killer instinct every emulator has a bug on the character select screen,hope someone make those 2 aswell.

**Comment by ulissesjdeo** [comment]
Hey guys, this project is using N64Recomp and already works
https://github.com/Francessco121/dino-recomp
